modules/apps/feature-flag/feature-flag-web/src/main/resources/META-INF/resources/js/FeatureFlagList.tsx

```diff
@@ -39,8 +39,8 @@ const FeatureFlagList: React.FC<IFeatureFlagListProps> = ({featureFlags}) => {
 							'no-feature-flags-were-found'
 						),
 					}}
-					imgSrc={`${Liferay.ThemeDisplay.getPathThemeImages()}/states/search_state.gif`}
-					imgSrcReducedMotion={`${Liferay.ThemeDisplay.getPathThemeImages()}/states/empty_state_reduced_motion.gif`}
+					imgSrc={`${Liferay.ThemeDisplay.getPathThemeImages()}/states/search_state.svg`}
+					imgSrcReducedMotion={`${Liferay.ThemeDisplay.getPathThemeImages()}/states/empty_state_reduced_motion.svg`}
```

Comment:
@ilzamcmed can you create a separate task to make sure we are using search_state_reduced_motion not empty_state_reduced_motion? Also, you will notice that in feature flags we never actually show the `empty_state_reduced_motion` even when reduced motion is on. We also need a new Clay ticket created to respect the `imgSrcReducedMotion` prop.